scripts/databricks_install.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# ---------------------------------------------------------
+# This script installs Recommenders into Azure Databricks
+
+echo "Preparing Recommenders library file (egg)..."
+zip -r -q Recommenders.egg . -i *.py -x tests/\* scripts/\*
+
+echo "Uploading to databricks..."
+dbfs cp --overwrite Recommenders.egg dbfs:/FileStore/Recommenders.egg
+
+# Cluster id should be passed by the first argument
+# Cluster id can be found in the URL at https://<databricks-instance>/?o=<16-digit-number>#/setting/clusters/$CLUSTER_ID/configuration.
```

Comment:
This should match the instructions in SETUP.md for retrieving the cluster id through the cli, doing this from the UI could break if databricks changes how the URL works.

Reply:
ha, though I see the databricks CLI provides the same suggestion... /shrug

Reply:
lol yeah. As using cli is easier, will update this to be the same as SETUP.md

Reply:
Updated the script. Replaced code comments to printing out messages.